lib/statement.rb

```diff
@@ -0,0 +1,27 @@
+
+class Statement
+  def initialize(transactions_log)
+    @transactions_log = transactions_log
+  end
+
+  def print_it
+    print_header
+    print_each_transaction
+  end
+
+  private
+
+  attr_reader :transactions_log
+
+  def print_header
+    puts ' date | credit | debit | balance '
+  end
+
+  def print_(transaction)
```

Comment:
Did you call this method `print_` so it reads in an English-like way with its `transaction` argument?